manticore/platforms/evm.py

```diff
@@ -21,10 +23,25 @@
 
 logger = logging.getLogger(__name__)
 
-#fixme make it global using this https://docs.python.org/3/library/configparser.html
-#and save it at the workspace so results are reproducible
-config = namedtuple("config", "out_of_gas")
-config.out_of_gas = None  # 0: default not enough gas, 1 default to always enough gas, 2: for on both
+#Gas behaivor configuration
+# When gas is concrete the gas checks and calculation are pretty straigth forward
+# Though Gas can became symbolic in normal bytecode execution for example at instructions
+# MSTORE, MSTORE8, EXP, ... and every instruction with internal operation restricted by gas
+# This configuration variable allows the user to control and perhaps relax the gas calculation
+# 0: gas is faithfully accounted and checked at instruction level. State may get forked in OOG/NoOOG
```

Comment:
Can we assign these options symbolic names, like `pedantic` (0), `complete` (1), `eager` (2), `optimistic` (3), `defer` (4) and `ignore` (99)? Afraid that if we store the command line with the saved flag, it's not very clear what the value was initially.